app/src/main/res/values/strings.xml

```diff
@@ -579,6 +579,15 @@
     <string name="minimize_on_exit_none_description">None</string>
     <string name="minimize_on_exit_background_description">Minimize to background player</string>
     <string name="minimize_on_exit_popup_description">Minimize to popup player</string>
+    <!-- Back button behaviour in a fragment -->
+    <string name="back_behavior_title">Back press in a player</string>
+    <string name="back_behavior_summary">Action when clicking on a back button on a player screen — %s</string>
+    <string name="back_behavior_prev_any_description">Open previous video from a history</string>
+    <string name="back_behavior_prev_playlist_description">Open previous video from a history (playlist only)</string>
+    <string name="back_behavior_hide_description">Hide into the mini player</string>
+    <string name="back_behavior_hide_or_close_description">Hide into the mini player if the stream was played, close otherwise</string>
```

Comment:
```suggestion
    <string name="back_behavior_hide_or_close_description">Minimize into the mini player if the stream is completed, close otherwise</string>
```
I am not sure I understood what this does exactly. If I understood correctly maybe this would be a better phrasing:
```suggestion
    <string name="back_behavior_hide_or_close_description">Close the video player if it is playing something, minimize into the mini player otherwise</string>
```